addons/mail/static/src/discuss/call/common/call.js

```diff
@@ -69,6 +70,15 @@ export class Call extends Component {
             browser.clearTimeout(this.overlayTimeout);
         });
         useExternalListener(browser, "fullscreenchange", this.onFullScreenChange);
+        useHotkey("shift+d", async () => {
+            this.rtc.selfSession.is_deaf ? this.rtc.undeafen() : this.rtc.deafen();
+            if (this.rtc.localSession.isMute) {
+                await this.rtc.toggleMicrophone();
+            }
+        });
```

Comment:
This should be a `toggleDeafen()`. I know the function doesn't exists but the code in `select` of the related call action `"deafen"` basically does the same logic as this `toggleDeafen()`.

The hotkey should map exactly the click on buttons, and to ensure reuse code prefer calling same function.